examples/combobox/combobox-select-only.html

```diff
@@ -0,0 +1,413 @@
+            </div>
+            <div class="combo-menu" role="listbox" id="listbox1" tabindex="-1">
+                <!-- options are inserted here -->
+                <!-- e.g. <div role="option" id="op1">option text</div>  -->
+            </div>
+        </div>
+      </div>
+      <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+    <section>
+      <h2 id="kbd_label">Keyboard Support</h2>
+      <p>
+      The example combobox on this page implements the following keyboard interface.
+        Other variations and options for the keyboard interface are described in the
+        <a href="../../#combobox_kbd_interaction">Keyboard Interaction section of the combobox design pattern.</a>
+      </p>
+      <h3 id="kbd_label_combobox">Closed Combobox</h3>
+      <table aria-labelledby="kbd_label_combobox kbd_label" class="def">
+        <thead>
+          <tr>
+            <th>Key</th>
+            <th>Function</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr data-test-id="combobox-key-down-arrow">
+            <th><kbd>Down Arrow</kbd></th>
+            <td>
+              <ul>
+                <li>Opens the listbox if it is not already displayed without moving focus or changing selection.</li>
+                <li>DOM focus remains on the combobox.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="combobox-key-alt-down-arrow">
+            <th><kbd>Alt + Down Arrow</kbd></th>
+            <td>
+                Opens the listbox without moving focus or changing selection.
+            </td>
+          </tr>
+          <tr data-test-id="combobox-key-up-arrow">
+            <th><kbd>Up Arrow</kbd></th>
+            <td>
+              <ul>
+                <li>First opens the listbox if it is not already displayed and then moves visual focus to the first option.</li>
+                <li>DOM focus remains on the combobox.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="combobox-key-enter">
+            <th><kbd>Enter</kbd></th>
+            <td>
+                Opens the listbox without moving focus or changing selection.
+            </td>
+          </tr>
+          <tr data-test-id="combobox-key-space">
+            <th><kbd>Space</kbd></th>
+            <td>
+                Opens the listbox without moving focus or changing selection.
+            </td>
+          </tr>
+          <tr data-test-id="combobox-key-home">
+            <th><kbd>Home</kbd></th>
+            <td>
+                Opens the listbox and moves visual focus to the first option.
+            </td>
+          </tr>
+          <tr data-test-id="combobox-key-end">
+            <th><kbd>End</kbd></th>
+            <td>
+                Opens the listbox and moves visual focus to the last option.
+            </td>
+          </tr>
+          <tr data-test-id="standard-single-line-editing-keys">
```

Comment:
might want to change this test id to "printable-characters"?

Reply:
actually it looks like this keyboard interaction is covered by tests with the `data-test-id="listbox-key-char"`. Right now, the regression report says there are no tests for this data-test-id but there are. @smhigley, maybe you should give both of these the same data-test-id, such as `printable-characters`, and update the test reference as well?